homeassistant/components/plugwise/climate.py

```diff
@@ -120,16 +120,6 @@ def hvac_mode(self) -> HVACMode:
     @property
     def hvac_action(self) -> HVACAction | None:
         """Return the current running hvac operation if supported."""
-        # When control_state is present, prefer this data
-        if (control_state := self.device.get("control_state")) == "cooling":
-            return HVACAction.COOLING
-        # Support preheating state as heating,
-        # until preheating is added as a separate state
-        if control_state in ["heating", "preheating"]:
-            return HVACAction.HEATING
```

Comment:
This removed the possibility to set `HVACAction.PREHEATING` as it seems?

Reply:
Yes, this specific info is lost to. So no longer possible to set `HVACAction.PREHEATING`, which we didn't have yet :)

Reply:
Right, but I think we should not loose it and instead reflect the state correctly.

Plugwise was one of the drivers to implement that core state to begin with.

Reply:
I will double-check once more later today on my Adam, just to be sure :)

Reply:
As discussed on Discord, data = gone in newer firmware. So, not preheating status possible. 👍